ui/src/app/welcome-page/welcome-page.component.html

```diff
@@ -0,0 +1,79 @@
+<div class="grid-row flex-column text-center border-top border-bottom">
+     <span class="background-image" role="img" aria-label="Beautiful green hills with big tree and mountains row on the horizon Free Vector
+     ">
+          <h3 class="grid-col flex-fill">Learn Programming Today</h3>
+          <button class="usa-button--secondary openmenu" (click)="sendEmitterForRegisterScreen('register')">Register</button>
+          <a class="grid-col flex-fill">Try As Guest</a>
+
+          <div class="characterSheet">
+               <!-- image container below: -->
+               <div class="welcomeScreenMonster1">
```

Comment:
There's got to be a better way to do this in Angular; needing to manually write out the monster parts is going to be an extreme pain in the rear. I can't imagine there's not a way to iterate through an object or something and place the tags appropriately.